src/precompiles/ArbNativeToken.sol

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021-2025, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro-contracts/blob/main/LICENSE.md
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity >=0.4.21 <0.9.0;
+
+/**
+ * @title Enables minting and burning native tokens.
+ * @notice Authorized callers are added/removed through ArbOwner precompile.
+ */
+interface ArbNativeToken {
```

Comment:
please add note on since which ArbOS version this precompile will be available

Reply:
Done.